app/helpers/layout_helper.rb

```diff
@@ -323,6 +323,12 @@ def trunc_with_tooltip(text, length = 32)
     content_tag(:span, truncate(text, :length => length), options).html_safe
   end
 
+  def text_with_tooltip(text, tooltip_text)
```

Comment:
maybe this could be combined with `trunc_with_tooltip`?
making tooltip_text the same as text if nothing was entered and adding the length to truncate?